index.js

```diff
@@ -146,8 +146,8 @@ exports.generate = function (opts, cb) {
                     // create path relative to opts.base
                     var relativeToBase = path.relative(path.resolve(opts.base), path.resolve(path.join(dir, filePath)));
 
-                    // prepend / to make it absolute
-                    return normalizePath(match.replace(filePath, path.join('/', relativeToBase)));
+                    var pathPrefix = (typeof opts.pathPrefix === "undefined") ? "/" : opts.pathPrefix;
+                    return normalizePath(match.replace(filePath, path.join(pathPrefix, relativeToBase)));
```

Comment:
That just won't work for me. I need it to be relative to the HTML file wherever it happens to be. It is in a different place in `dev` and in `qa` and in `prod`. Otherwise I'll have to have specific critical tasks for each of those places.

Also, I was unaware you could have more than one `dest`? Unless I'm misunderstanding why this is a big deal... If I'm understanding you correctly, if you target two different `dest` files for output and they are in two different directory depths then (of course) this is going to screw up because the `pathPrefix` will only be valid for _one_ of those two outputs. But, if it is just a single `dest` file, then, why would it matter if this was relative instead of absolute?